data/templates/metronome/domain.tpl.cfg.lua

```diff
@@ -70,6 +70,84 @@ Component "xmpp-upload.{{ domain }}" "http_upload"
   http_upload_file_size_limit = 100 * 1024 * 1024 -- bytes
   http_upload_quota = 10 * 1024 * 1024 * 1024 -- bytes
 
+  http_file_allowed_mime_types = {
+      ["3gp"] = "video/3gpp",
+      ["7z"] = "application/x-7z-compressed",
+      ["aac"] = "audio/aac",
+      ["abw"] = "application/x-abiword",
+      ["apk"] = "application/vnd.android.package-archive",
+      ["arc"] = "application/octet-stream",
+      ["avi"] = "video/x-msvideo",
+      ["azw"] = "application/vnd.amazon.ebook",
+      ["bin"] = "application/octet-stream",
+      ["bmp"] = "image/bmp",
+      ["bz2"] = "application/x-bzip2",
+      ["bz"] = "application/x-bzip",
+      ["csh"] = "application/x-csh",
+      ["css"] = "text/css",
+      ["csv"] = "text/csv",
+      ["doc"] = "application/msword",
+      ["docx"] = "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
+      ["eot"] = "application/vnd.ms-fontobject",
+      ["epub"] = "application/epub+zip",
+      ["gif"] = "image/gif",
+      ["gz"] = "application/x-compressed",
+      ["html"] = "text/html",
```

Comment:
Wouldn't allowing HTML, XHTML and SVG open the server to [XSS from users](https://xmpp.org/extensions/xep-0363.html#server)?